stubs/chevron/chevron/renderer.pyi

```diff
@@ -0,0 +1,19 @@
+from _typeshed import StrOrBytesPath, SupportsRead
+from typing import Any, Sequence
+
+from tokenizer import tokenize as tokenize
+
+g_token_cache: dict[str, list[tuple[str, str]]]  # undocumented
+
+def render(
+    template: SupportsRead[str] | str = ...,
+    data: dict[str, Any] = ...,
+    partials_path: StrOrBytesPath | None = ...,
+    partials_ext: str = ...,
+    partials_dict: dict[str, str] = ...,
+    padding: str = ...,
+    def_ldel: str | None = ...,
+    def_rdel: str | None = ...,
```

Comment:
Does `None` really work? The default values are strings, and the docstring says nothing about `None`s.

Reply:
Yes, `l_del` and `r_del` are used in `tokenizer.py` as `.split(X_del, ...)` and in string comparison.
```python
>>> import chevron
>>> chevron.render('Hello', {'Hello': 'World'}, def_ldel=None, def_rdel=None)
'Hello'
```